testutils/src/main/java/com/amplifyframework/testutils/Assets.java

```diff
@@ -51,8 +51,11 @@ public static String readAsString(final String name) throws RuntimeException {
 
             String line = bufferedReader.readLine();
             while (line != null) {
-                stringBuilder.append(line).append("\n");
+                stringBuilder.append(line);
                 line = bufferedReader.readLine();
+                if (line != null) {
+                    stringBuilder.append("/n");
```

Comment:
I think you meant to keep this as `\n`, not `/n`.

Reply:
Ahhh, you are absolutely right! That's an embarrassing mistake. Thanks for catching it